yang/frr-vrf.yang

```diff
@@ -36,29 +40,32 @@ module frr-vrf {
           "VRF name.";
       }
 
-      leaf id {
-        type uint32 {
-          range "0..4294967295";
+      container netns {
+        /* FIXME: libyang support is required */
+        /* if-feature "netns"; */
+        leaf name {
+          type string;
+          description
+            "Namespace name.";
         }
-        config false;
-        description
-          "VRF Id.";
       }
 
-      leaf active {
-        type boolean;
-        default "false";
+      container state {
         config false;
-        description
-          "VRF active in kernel.";
-      }
+        leaf id {
+          type uint32 {
+            range "0..4294967295";
+          }
+          description
+            "VRF Id.";
+        }
 
-      container netns {
-        if-feature "netns";
-        leaf name {
-          type string;
+        leaf active {
+          type boolean;
+          default "false";
+          config false;
```

Comment:
This is unnecessary. When not specified, the "config" setting is inherited from the parent node (which is "state" in this case).

Reply:
Sure will remove it.